adserver/templates/adserver/email/base.html

```diff
@@ -0,0 +1,18 @@
+{# Email intentionally uses an old doctype for maximum compatibility #}
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="https://www.w3.org/1999/xhtml">
+<head>
+  {# This title is never displayed but there's a block in case you want to customize it #}
```

Comment:
Don't quite follow this. Will it be displayed if it's customized?

Reply:
No, it will not. The `<title>` tag in an HTML is never seen unless somebody looks at the original source of the email.